astrosat_users/adapters.py

```diff
@@ -297,6 +297,14 @@ def send_password_confirmation_email(self, user, email, **kwargs):
     def send_mail(self, template_prefix, email, context, **kwargs):
         cc = kwargs.pop("cc", [])
         bcc = kwargs.pop("bcc", [])
+        if cc:
+            if isinstance(cc, str):
+                raise TypeError('"cc" argument must be a list or tuple')
+            cc = list(cc)
+        if bcc:
+            if isinstance(bcc, str):
+                raise TypeError('"bcc" argument must be a list or tuple')
+            self.bcc = list(bcc)
```

Comment:
I assume this is to convert a `tuple` to a `list`, if it is already a `list`, then re-wrapping it as a list, is little extra effort.

Reply:
It converts anything that isn't a `str` (or `None` or the empty list) to a `list`.  I just copied this logic from Django itself: https://github.com/django/django/blob/755dbf39fcdc491fe9b588358303e259c7750be4/django/core/mail/message.py#L207-L210